docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -123,21 +123,46 @@ strategy has certain limitations:
 We highly recommend leaving `readOnly` unset or explicitly setting it to
 `false`.
 
+## Partitioning
+
+[Partitioning][wiki-partitioning] is an extremely effective optimization for
+improving data access. It effectively "shards" the data between multiple tables,
+splitting them by a defined attribute. An incoming query would be checked for
+this attribute, and any valid partitions required to satisfy it are selected.
+This results in faster refresh times and possibly reduced cost, depending on
+your database.
+
+Cube.js supports [both time][ref-preagg-time-part] and
+[segment-based][ref-preagg-segment-part] partitioning. However, it first must be
+enabled for each pre-aggregation.
+
+[Time-based partitioning][ref-preagg-time-part] is especially helpful for
+incremental refreshes; when configured, Cube.js will only refresh partitions as
+necessary. Without incremental refreshing, Cube.js will re-calculate the entire
+pre-aggregation whenever [the refresh key][ref-preaggs-refresh-key] changes.
+
 ## Pre-Aggregations Storage
 
-When using **external** pre-aggregations, Cube.js will
-store pre-aggregations inside its own purpose-built storage layer: Cube Store.
+When using **external** pre-aggregations, Cube.js will store pre-aggregations
+inside its own purpose-built storage layer: Cube Store.
```

Comment:
add reference to the Cube Store doc?

Reply:
We don't have any Cube Store docs available right now - coming soon 👀